Makefile

```diff
@@ -697,14 +697,22 @@ nova_deploy_cleanup: ## cleans up the service instance, Does not affect the oper
 ##@ KUTTL tests
 
 .PHONY: mariadb_kuttl
-mariadb_kuttl: namespace input openstack_crds deploy_cleanup mariadb_deploy_prep mariadb  ## runs kuttl tests for the mariadb operator. Installs openstack crds and keystone operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+mariadb_kuttl: input ## runs kuttl tests for the mariadb operator, assumes that everything needed for running the test was deployed beforehand.
```

Comment:
lgtm. just a thought on the name, perhaps it is better to keep mariadb_kuttl as the 'do everything target' and have a special one for devs that just want the tests like "mariadb_kuttl_only"  so it is more descriptive

Reply:
Yeah, maybe that makes more sense, changing